ibis/backends/tests/test_client.py

```diff
@@ -1764,3 +1764,8 @@ def test_insert_using_col_name_not_position(con, first_row, second_row, monkeypa
     # Ideally we'd use a temp table for this test, but several backends don't
     # support them and it's nice to know that data are being inserted correctly.
     con.drop_table(table_name)
+
+
+def test_from_dbapi_connection(con):
```

Comment:
Ideally we can run the entire test suite using a backend constructed this way, which will give us more confidence that the API works.

That can be done in a follow up though.

Reply:
How do you envision doing this without running the whole test suite again? Or it's fine to run it again?

Reply:
You can (somewhat arbitrarily) pick a backend and modify its `conftest.py` to use this way of initialization.

Reply:
But then we wouldn't have the other way of initialization, right? Or, because there's `ddl_con` and `con` fixtures both, it would be OK?

Will look to explore this in a followup then; will just get the rest of the backends added and tests passing here first.